src/screens/SwapScreen/ActionModal.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { useTheme } from "@emotion/react";
+import styled from "@emotion/styled";
+
+import { ModalEnums } from "@screens/SwapScreen/enums/modalEnums.tsx";
```

Comment:
```suggestion
import { ModalEnums } from "@screens/SwapScreen/enums/modalEnums";
```

Думаю надо убрать tsx в конце

Reply:
Исправил